nodejs_conductor/native/src/config.rs

```diff
@@ -83,11 +85,13 @@ fn make_config(instance_data: Vec<InstanceData>, logger: LoggerConfiguration) ->
 
         let agent_id = agent_config.id.clone();
         let dna_id = dna_config.id.clone();
+        let temp = tempdir().expect("test was supposed to create temp dir");
```

Comment:
I was looking up [tempdir's docs](https://docs.rs/tempfile/3.0.7/tempfile/fn.tempdir.html) because I was wondering if that is a unique directory since we don't want multiple test instances share the same CAS. It says that the tempdir gets removed when `TempDir` is dropped. This happens a few lines below, right? The conductor will recreate the the path I guess but then nothing will delete it, so it actually is not temporary. This might cause mysterious problems later on.. Not sure if we actually want to merge pickle as default for nodejs conductor into develop.. We might want to if it really is faster than MemoryStorage..

Reply:
I agree, I think since memory is the fastest and this is for the use in the nodejs we should still go back to memory. I will revert this back asap